pages/logout.page.js

```diff
@@ -0,0 +1,20 @@
+
+
+import BasePage from '../pages/base.page';
+
+class LogoutPage extends BasePage {
+    //WebElements
+    get userAvatar(){return $('[class="user-avatar rounded-circle"]')};
```

Comment:
Prefer using only `user-avatar` class, if that's enough for locating that element, as `rounded-circle` seems to be a more general class, used exclusively for styling purposes, and can potentially change faster than user-avatar (if they decide to style it differently, for example), making your selector more flaky / less robust.

Reply:
![image](https://user-images.githubusercontent.com/96138690/147512033-7d6ccebc-c3e4-4ac2-9f5b-62bdaf29825f.png)
that's the full name of the class...

Reply:
Those are 2 classes (multiple css classes adding in html elements is achieved by separating each one with an empty space, so you add classes to an element by adding them after an empty space). If only one html element has one class, you can still find it by targeting that specific class with a css selector (like in this case, **user-avatar**)

Reply:
Error:
element ("[class="user-avatar"]") still not clickable after 10000ms

Reply:
what if you try a raw css selector (like `.user-avatar`) like:
```javascript
$(".user-avatar")
```

Reply:
Done:
get userAvatar(){return $('.user-avatar');}